index.html

```diff
@@ -0,0 +1,109 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main id="main">
+          <h1 id="title">Dr Norman</h1>
+          <div id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"/>
```

Comment:
Add "alt" attribute to your image tag. You image might not shows in some instances. It is a good practise to use alternative text to in your image tag. This tells the user/reader what the image is all about.